examples/simple/simple.go

```diff
@@ -25,46 +25,87 @@ func New(logger logger.Logger) *Handler {
 
 // AuthConnect is called on device connection,
 // prior forwarding to the MQTT broker
-func (h *Handler) AuthConnect(ctx context.Context, c *session.Client) error {
+func (h *Handler) AuthConnect(ctx context.Context) error {
+	var c session.Client
+	if err := c.FromContext(ctx); err != nil {
+		h.logger.Error("client is missing: " + err.Error())
```

Comment:
Already err implies client is missing
```suggestion
		h.logger.Error(err.Error())
```

Reply:
not yet addressed